docs/modules/ROOT/pages/tmail-backend/jmap-extensions/calendarEventParsing.adoc

```diff
@@ -0,0 +1,137 @@
+to advertise it through the session capabilities:
+....
+com:linagora:params:calendar:event
+....
+
+The associated object is empty.
+
+== CalendarEvent/parse
+
+This method bases on:
+- Parsing a Received MDN method as described in the RFC-9007
+- JMAP CalendarEvent RFC-8620
+- iCalendar message RFC-5545
+
+The request object requires the following properties:
+
+- `accountId`: accountId of action user
+- `blobIds`: an array blobId of calendar which user wants to parse.
+
+The response object contains the following properties:
+
+- `accountId`: accountId of action user
+- `parsed`: a map of blobId and the corresponding calendar parse result.
+- `notFound`: an array of blobId is not found.
+- `notParsable`: an array of blobId has been found but is not parsable.
+
+== Example
+
+The client's request to parse the calendar event which has blobId is `0f9f65ab-dc7b-4146-850f-6e4881093965`
+
+....
+{
+	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:calendar:event"],
+	"methodCalls": [
+        [ "CalendarEvent/parse", {
+            "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+            "blobIds": ["0f9f65ab-dc7b-4146-850f-6e4881093965"]
+        }, "c1"]
+	]
+}
+....
+
+The server responds:
+
+```
+[[ "CalendarEvent/parse",
+{
+    "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+    "parsed": {
+        "0f9f65ab-dc7b-4146-850f-6e4881093965": {
+            "title": "Gatling: break LemonLDAP!",
+            "description": "Let's write some basic OIDC benchmarks",
+            "start": "2023-02-09T10:00:00",
+            "duration": "PT2H0M0S",
+            "end": "2023-02-09T12:00:00",
+            "timeZone": "Asia/Ho_Chi_Minh",
+            "location": "5 Dien Bien Phu, Ha Noi",
+            "organizer": {
+                "name": "Benoît TELLIER",
+                "mailto": "btellier@linagora.com"
+            },
+            "participants": [{
+                    "name": "Benoît TELLIER",
+                    "mailto": "btellier@domain.tld",
+                    "kind": "individual",
+                    "role": "chair",
+                    "participationStatus": "accepted",
+                    "expectReply": false
+                },
+                {
+                    "name": "Van Tung TRAN",
+                    "mailto": "vttran@domain.tld",
+                    "kind": "individual",
+                    "role": "requested-participant",
+                    "participationStatus": "needs-action",
+                    "expectReply": true
+                }
+            ],
+            "extensionFields": {
+                "X-OPENPAAS-VIDEOCONFERENCE": "https://jitsi.linagora.com/abcd",
+                "X-OPENPAAS-CUSTOM-HEADER1": "whatever"
+            }
+        }
+    }
+}, "c1" ]]
+```
+
+The `parsed` value contains the following properties:
+
+- `title`: title of the event ~ SUMMARY (iCalendar)
```

Comment:
We should precise that the properties match with the properties defined in rfc5545 (I mean SUMMARY, DURATION, DTSTART, etc). It's not necessary obvious reading it just like that